Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -264,6 +264,9 @@ public class LocalStorage {
         try createDirectory(at: submittedJobDirectory)
         let unsubmittedFileDirectory = try unsubmittedJobDirectory.appendingPathComponent(jobId)
         let submittedFileDirectory = try submittedJobDirectory.appendingPathComponent(jobId)
+        if fileManager.fileExists(atPath: submittedFileDirectory.relativePath) {
+            try fileManager.removeItem(atPath: submittedFileDirectory.relativePath)
```

Comment:
Can you add a log here, since it's a destructive action?

Reply:
the func "throws" so that would be printed where we call this `moveToSubmittedJobs` func

I don't think we need an extra log here